src/App.js

```diff
@@ -1,9 +1,30 @@
+import React from "react";
+import { BrowserRouter as Router, Route, Link, Switch } from "react-router-dom";
+import list from "./Components/list";
```

Comment:
Make sure you capitalize component names
```suggestion
import List from "./Components/list";
```

Reply:
Also, don't commit this suggestion because below in this file there are two usages with `list` instead of `List`. It'll break 🙀